src/errors.rs

```diff
@@ -15,6 +15,8 @@ pub enum Error {
         error_link: String,
     },
 
+    /// The MeiliSearch request timed out.
+    MeiliSearchTimeoutError { message: String },
```

Comment:
I don't think it's a good idea to add a new variant in this case since there is only one constructor on this variant.